Lanchonete/Lanchonete/ Features/Confirmation/Presentation/ConfirmationViewPresenter.swift

```diff
@@ -43,4 +54,15 @@ class ConfirmationViewPresenter {
     func getSandwichIngredients() -> [Ingredient] {
         return sandwichManager.getIngredients()
     }
+    
+    func addToShoppingCart() {
```

Comment:
There are no tests for this method.

Reply:
The only difference between this method and the addSandwichWithQuantity is the delegate action. I won't be adding tests for this method since I added tests for addSandwichWithQuantity